### PR TITLE
MM-41865 Creating a new explore for Calls Plugin Telemetry

### PR DESCRIPTION
**Pull request by sri-byte**
1) Impact; This is new explore for Calls Plugin Telemetry. Creating this explore from the events schema. From this new explore we will be able to create new dashboards for Calls Plugin Telemetry data.
2) Testing: The data has been loaded into the database manually, once this PR is approved, I will build the dashboard and get it reviewed by the Calls team and create a separate PR for the backend code.

### CODE REVIEW

File: views/mattermost/server_fact.view.lkml
Comment (jasonblais) [inline views/mattermost/server_fact.view.lkml:2]: Are changes in this file intended for this PR?

**Review by jasonblais — COMMENTED** [review]
Just one question below, thanks Sri!

File: views/mattermost/server_fact.view.lkml
Comment (sri-byte) [reply views/mattermost/server_fact.view.lkml:2]: You're right @jasonblais, I thought the changes would create a new PR, I removed these changes from this one.

Comment (sri-byte) [reply views/mattermost/server_fact.view.lkml:2]: I'll create a new one for these changes, once this PR gets merged.

Comment (jasonblais) [reply views/mattermost/server_fact.view.lkml:2]: @sri-byte If you create a new git branch and submit your changes for server_fact table there, you can submit a separate PR for this second git branch. That way you don't have to wait until this PR is merged. I can simply approve the one for the server_fact table.

Let me know if you have any questions about using multiple git branches for separate PRs.

File: views/events/calls_events.view.lkml
Comment (catalintomai) [inline views/events/calls_events.view.lkml:9]: these look to be generic comments (are they coming from the template?), should we remove them?

File: views/events/calls_events.view.lkml
Comment (catalintomai) [inline views/events/calls_events.view.lkml:18]: I would remove the first two rows.

File: views/events/calls_events.view.lkml
Comment (catalintomai) [inline views/events/calls_events.view.lkml:93]: remove?

File: views/mattermost/server_fact.view.lkml
Comment (catalintomai) [inline views/mattermost/server_fact.view.lkml:161]: @sri-byte , please fix the spacing to clearly delineate what are the intended changes. Thanks!.

**Review by streamer45 — APPROVED** [review]
Looks good from a data perspective, everything we currently track is here, thanks @sri-byte :+1:

**Review by itao — APPROVED** [review]

File: views/events/calls_events.view.lkml
Comment (catalintomai) [reply views/events/calls_events.view.lkml:18]: I'm mostly referring to comments like "Here's what a typical dimension looks like in LookML.", I assume they are canned comments coming from some template that can be removed and should add instead more useful/topical ones.

**Pull request merged**
